src/restconfig/src/test/java/org/geoserver/rest/system/status/MonitorRestTest.java

```diff
@@ -69,6 +74,41 @@ public void testJsonCallback() throws Exception {
         assertTrue(metrics.getMetrics().size() >= MetricInfo.values().length);
     }
 
+    @Test
+    public void testJsonValueFieldIsEncoded() throws Exception {
+        MockHttpServletResponse response =
+                getAsServletResponse(RestBaseController.ROOT_PATH + "/about/system-status.json");
+        assertEquals(200, response.getStatus());
+        assertEquals("application/json", response.getContentType());
+        JSONArray array =
+                ((JSONObject)
+                                ((JSONObject) JSONSerializer.toJSON(response.getContentAsString()))
+                                        .get("metrics"))
+                        .getJSONArray("metric");
+        assertTrue(array.size() == MetricInfo.values().length);
+        for (int i = 0; i < array.size(); i++) {
```

Comment:
If my memory serves me well, if a metric is not available in a certain environment (for example the machine doesn't provide a network sensor) the the metric will not be collected. Hence, checking that available metrics have a value makes sense, but checking the size probably not :scream:.

Reply:
Thanks for your review @nmco, this has  been fixed

Reply:
Great, thx!